inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/convert.hpp

```diff
@@ -23,13 +23,14 @@ using ConvertParamsTuple = typename std::tuple<
         InferenceEngine::Layout,           // Output layout
         std::string>;                      // Device name
 
-class ConvertLayerTest : public testing::WithParamInterface<ConvertParamsTuple>,
-                        virtual public LayerTestsUtils::LayerTestsCommon {
+class NGRAPH_DEPRECATED("This class is deprecated and will be removed soon.") ConvertLayerTest :
```

Comment:
In a perfect world, we should mention to use new common ConversionLayerTest class instead:))

Reply:
Let's never forget that we should strive for perfection 😄 Updated the macro message just as suggested.